src/components/common/Select/select.css

```diff
@@ -109,11 +109,17 @@
     margin-bottom: -8px;
     box-shadow: 0 -1px 0 0 var(--N200);
     background-color: inherit;
-    padding: 10px 12px;
+    padding: 11px 12px;
     width: 100%;
     display: flex;
     align-items: center;
     cursor: pointer;
+    color: var(--B500);
```

Comment:
is it intentional? it will reflect in all the places it is used

Reply:
Yes. We wanted to change the color across tabs/sections.